backend/src/test/resources/graphql-test/add-multiplex-result-mutation.graphql

```diff
@@ -0,0 +1,17 @@
+mutation addMultiplexResult($deviceId: String!, $results: [MultiplexResultInput]!, $patientId: ID!, $dateTested: DateTime) {
```

Comment:
This is not a new file, it was just moved to `backend/src/test/resources/graphql-test` dir, git is not showing that correctly